ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -494,6 +340,36 @@ open class SegmentedControl: UIControl {
         return buttons[index] as UIView
     }
 
+    var defaultTokens: SegmentedControlTokens = .init()
+    var tokens: SegmentedControlTokens = .init()
+    var overrideTokens: SegmentedControlTokens? {
+        didSet {
+            updateSegmentedControlTokens()
+            updateColors()
+            updateButtons()
+        }
+    }
+
+    var selectionChangeAnimationDuration: TimeInterval { return 0.2 }
+
+    private func updateSegmentedControlTokens() {
+        let tokens = resolvedTokens
+        tokens.style = style
+        self.tokens = tokens
+    }
+
+    private func updateButtons() {
+        for (index, button) in buttons.enumerated() {
+            button.tokens = tokens
+            if isEnabled {
```

Comment:
nit: since the 2 calls are identical with only the different param, it could be cleaner to just create a `let labelColor` and assign it with the correct color based on the isEnabled and just have one call in the code to set the title color on the button.